src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -234,6 +240,38 @@ const Actions: React.FC<Props> = ({
         };
     };
 
+    const mapsLinkForParcels = (): string => {
+        const mapsLinkList = ["https://www.google.com/maps/dir/"];
+        selectedParcels.map((parcel) => {
+            mapsLinkList.push(parcel.addressPostcode + "/");
+        });
+        mapsLinkList.push("/");
+        const mapsLinkString = mapsLinkList.join("");
+        return mapsLinkString;
+    };
+
+    const openInNewTab = (url: string): void => {
+        const newWindow = window.open(url, "_blank", "noopener, noreferrer");
+        if (newWindow) {
+            newWindow.opener = null;
+        }
```

Comment:
I'm not sure I understand this if statement, could you please explain? :)

Reply:
newWindow is opening a page with the specified url in a new tab(_blank).
noopener:  to prevent an external link from taking control of your user’s originating browser window (this is handled by the major browsers but not android web view - so unsure if we need this anymore)
noreferrer:  keeps external sites from knowing that you have included links to their material on your website
newWindow.opener = null: To set the opener to null again (again to stop the passage into our site - but I'm not sure if its outdated.

Reply:
Ok cool, if you think its needed, then looks good :)